docker/Dockerfile-rocm

```diff
@@ -1,6 +1,6 @@
-FROM rocm/dev-ubuntu-18.04:3.0
+FROM rocm/dev-ubuntu-18.04:latest
```

Comment:
I'd prefer to pin this to 3.3 so that it's clear which base image is used in that dockerfile.

Reply:
Then we need to put some procedure in place to regularly test against latest so we see when new versions break things. We should probably do the same for CUDA, where we tend to be at least 6 months behind on testing and support.